plugin-flex-ts-template-v2/src/feature-library/park-interaction/custom-components/ParkView/ParkViewTable.tsx

```diff
@@ -0,0 +1,86 @@
+import { TBody, THead, Table, Td, Th, Tr, Button } from '@twilio-paste/core';
+import { ChatIcon } from '@twilio-paste/icons/esm/ChatIcon';
+import { Actions, Manager, templates } from '@twilio/flex-ui';
+import { useState } from 'react';
+
+import { StringTemplates } from '../../flex-hooks/strings';
+import ParkViewTableBodyWrapper from './ParkViewTableBodyWrapper';
+
+interface ParkViewTableProps {
+  recentInteractionsList: Array<any>;
+  isLoaded: boolean;
+}
+
+interface ParkViewItem {
+  key: string;
+  channel: string;
+  address: string;
+  customerName: string;
+  parkingDate: Date;
+  webhookSid: string;
+}
+
+const ParkViewTable = (props: ParkViewTableProps) => {
+  const [isUnparkingSid, setIsUnparkingSid] = useState('');
+  const instanceLanguage = Manager?.getInstance().configuration.language || 'en-US';
+
+  const resumeInteraction = async (ConversationSid: string, WebhookSid: string) => {
+    setIsUnparkingSid(ConversationSid);
+    try {
+      await Actions.invokeAction('UnparkInteraction', { ConversationSid, WebhookSid });
```

Comment:
Could we reload the table after this action completes? I noticed that I can keep clicking the unpark button while the notification banner is showing, and that causes me to get multiple tasks for the same conversation.

Reply:
Just included this! Please take a look when you can.